### PR TITLE
Add signatory organization info

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
ECOM-4009

@awais786 @tasawernawaz please review.

### CODE REVIEW

**Comment by awais786** [comment]
Please add a test case and check the value in response.

**Comment by awais786** [comment]
:+1:

**Comment by ahsan-ul-haq** [comment]
@jimabramson @clintonb please review.

**Comment by jimabramson** [comment]
@ahsan-ul-haq :+1:  to these changes.  Optional (not part of this change) could you take a quick look at the tests using `self._assert_signatory_data` - these seem like they might be misleading because in the `setUp`, `signatory_1` and `signatory_2` have the same values.

File: credentials/apps/credentials/models.py [hunk 1]
```diff
@@ -88,6 +88,10 @@ class Signatory(TimeStampedModel):
     """
     name = models.CharField(max_length=255)
     title = models.CharField(max_length=255)
+    organization_name = models.CharField(
```
Comment (clintonb) [inline credentials/apps/credentials/models.py]: I would call this `organization_name_override` to be completely explicit about the fact that this is not generally the normal use case.

File: credentials/templates/credentials/program_certificate.html [hunk 2]
```diff
@@ -115,7 +115,11 @@ <h4 class="signatory-name">{{ signatory.name }}</h4>
 
                                         <p class="signatory-credentials">
                                             <span class="role">{{ signatory.title }}</span>
-                                            <span class="organization">{{ certificate_context.organization_data.name }}</span>
+                                            {% if signatory.organization_name %}
```
Comment (clintonb) [inline credentials/templates/credentials/program_certificate.html]: Use [`firstof`](https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#firstof).

**Comment by clintonb** [comment]
:+1: once comments addressed

**Pull request merged**